sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-helper.ts

```diff
@@ -4,10 +4,35 @@ import {
   DATE_FORMAT,
   TIME_FORMAT,
 } from "@sims/integrations/esdc-integration/msfaa-integration/models/msfaa-integration.model";
-import { OfferingIntensity } from "@sims/sims-db";
+import { OfferingIntensity, RelationshipStatus } from "@sims/sims-db";
 import { getISODateOnlyString } from "@sims/utilities";
 import * as dayjs from "dayjs";
 
+export const THROW_AWAY_MSFAA_NUMBER = "3000";
+
+/**
+ * Values part of an MSFAA file record.
+ */
+export interface MSFAATestInputData {
```

Comment:
Why this interface was moved?

Reply:
I have put the interface back in the same model file as before and just renamed the file to be generic for both fulltime and part time.